streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/graph/ExecutionNode.java

```diff
@@ -79,6 +80,14 @@ public void setStreamOperator(StreamOperator streamOperator) {
     this.streamOperator = streamOperator;
   }
 
+  public void setLanguage(Language language) {
```

Comment:
Language should be calculated from the streamOperator.  Again, please consider add language property into the Operator or at least the streamOperator interface.

Reply:
good catch, just added